docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,323 @@
+        session-queue-size: 100 # A size of how many sessions will be handled. default to 100.
+    
+    edgehub:
+        websocket:
+            url: wss://0.0.0.0:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events 
+            certfile: /etc/kubeedge/certs/edge.crt
+            keyfile: /etc/kubeedge/certs/edge.key
+            handshake-timeout: 30 #second
+            write-deadline: 15 # second
+            read-deadline: 15 # second
+        quic:
+            url: 127.0.0.1:10001
+            cafile: /etc/kubeedge/ca/rootCA.crt
+            certfile: /etc/kubeedge/certs/edge.crt
+            keyfile: /etc/kubeedge/certs/edge.key
+            handshake-timeout: 30 #second
+            write-deadline: 15 # second
+            read-deadline: 15 # second
+        controller:
+            protocol: websocket # websocket, quic
+            heartbeat: 15  # second
+            project-id: e632aba927ea4ac2b575ec1603d56f10
+            node-id: edge-node
+    
+    edged:
+        register-node-namespace: default
+        hostname-override: edge-node
+        interface-name: eth0
+        edged-memory-capacity-bytes: 7852396000
+        node-status-update-frequency: 10 # second
+        device-plugin-enabled: false
+        gpu-plugin-enabled: false
+        image-gc-high-threshold: 80 # percent
+        image-gc-low-threshold: 40 # percent
+        maximum-dead-containers-per-container: 1
+        docker-address: unix:///var/run/docker.sock
+        runtime-type: docker
+        remote-runtime-endpoint: unix:///var/run/dockershim.sock
+        remote-image-endpoint: unix:///var/run/dockershim.sock
+        runtime-request-timeout: 2
+        podsandbox-image: kubeedge/pause:3.1 # kubeedge/pause:3.1 for x86 arch , kubeedge/pause-arm:3.1 for arm arch, kubeedge/pause-arm64 for arm64 arch
+        image-pull-progress-deadline: 60 # second
+        cgroup-driver: cgroupfs
+        node-ip: ""
+        cluster-dns: ""
+        cluster-domain: ""
+        
+        mesh:
+            loadbalance:
+                strategy-name: RoundRobin
+```
+
+#### Modification in edge.yaml
+
+1. Update the IP address of the master in the `edgehub.websocket.url` field. You need set cloudcore ip address.
+
+    ```yaml
+    url: wss://0.0.0.0:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events
+    ```
+    to 
+
+    ```yaml
+    url: wss://<X.X.X.X>:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events
+    ```
+    where X.X.X.X is your cloudcore IP Address.
+
+2. Update the IP address of the master in the `edgehub.quic.url` field. You need set cloudcore ip address.
+
+    ```yaml
+    quic:
+        url: 127.0.0.1:10001
+    ```
+    to 
+    ```yaml
+    quic:
+        url:<X.X.X.X>:10001
+    ```
+3. Check `edged.podsandbox-image` : This is very important and must be set correctly.
+
+   To check the architecture of your machine run the following
+
+    ```
+    getconf LONG_BIT
+    ```
+
+    + `kubeedge/pause-arm:3.1` for arm arch
```

Comment:
@chendave Just wanted to confirm on this. Should it be arm32? In the configuration file https://raw.githubusercontent.com/kubeedge/kubeedge/master/edge/conf/edge.yaml provided on github. It says `kubeedge/pause-arm:3.1`

Reply:
@chendave Have pushed the changes suggested (except this one). Let me know if you are fine with the changes.

Reply:
I am fine with this, what I meant was it might be something like this, "`kubeedge/pause-arm:3.1` for arm32 arch" to differentiate with aarch64 arch.  No need to respin.